client/src/test/java/model/BoundariesPositionTest.java

```diff
@@ -0,0 +1,7 @@
+package model;
+
+import static org.testng.Assert.*;
+
+public class BoundariesPositionTest {
```

Comment:
do not push empty class

Reply:
I dont know why it has been pushed :/

Reply:
ok

Reply:
It shouldnt be pushed :/ I add one test to fill it with something